### DISCUSSION TITLE
Mimic HA light icon colors

### DISCUSSION THREAD

**Discussion by skynetforcats** [Mariusthvdb/custom-ui#62] [Q&A]
Any idea how to template the exact same colors as lights, but just for some other entities?
I assume they are calculated somehow based on brightness percentage.

**Comment by Mariusthvdb** [comment]
this is a bit short on info, to even understand what your goal is...
what do you want to customize, based on what?

**Reply by skynetforcats** [reply]
Ok so it is for a fan entity. Where I wanted to use the same colors as lights use. Going from 1% = that brownish color to 100% bright yellow.

**Reply by Mariusthvdb** [reply]
yes, you can write a template for that, using any granularity you desire. check the examples for some ideas

**Reply by Mariusthvdb** [reply]
locking this, as it has been a while, and no additional info provided.
